classes/abicheck.bbclass

```diff
@@ -49,6 +49,22 @@ python binary_audit_abixml_compare_to_ref() {
     t0 = time.monotonic()
 
     pn = d.getVar("PN")
+    
+    util.note("FILESPATH {}".format(d.getVar("FILESPATH")))
+    
+    util.note("WORKDIR {}".format(d.getVar("WORKDIR")))
+    recipe_suppr = d.getVar("WORKDIR") + "/*_recipe.suppr"
```

Comment:
There should be some more clear name convention. Perhaps start with `abi`, like `abi*.suppr`? Just to ensure there are possibly no collisions with unrelated files. Then yeah, could support multiple suppression files as well.

Thanks